array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,10 @@
+/* exported tail */
+function tail(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== 0) {
```

Comment:
Same here, just start your loop one index later, and you can avoid the conditional